addons/website/static/src/js/widgets/theme.js

```diff
@@ -508,6 +512,12 @@ var ThemeCustomizeDialog = Dialog.extend({
             return self._quickEdit($(inputData));
         }));
 
+        // Handle auto changes
+        var $inputsAuto = $options.find('[data-widget="auto"]');
```

Comment:
- Search by classname
- As said, only take the checked ones or the alone ones, here you are calling `_setAuto` on all auto widgets.